### PR TITLE
add chatie landing page

### PR DESCRIPTION
**Pull request by lijiarui**
preview: https://lijiarui.github.io/chatie-coming-soon/index.html

### CODE REVIEW

**Comment by lijiarui** [comment]
Sorry to confuse you.

Make it work:

1. Add github page for master branch.
2. Change id for mailchimp as I show above.

Btw, do you know how to get mailchimp form id? If not, l will add information for mailchimp

File: index.html [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            <div>
+              <form action="https://botorange.us19.list-manage.com/subscribe/post?u=5277a9fd15328c03841df5827&amp;id=e493e654e5" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+                <div id="mc_embed_signup_scroll" class="form-row">
+
+                  <div class="col-12 col-md-9 mb-2 mb-md-0">
+                    <input type="email" value="" name="EMAIL" class="form-control form-control-lg required email" id="mce-EMAIL" placeholder="Enter your email...">
+                  </div>
+
+                  <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+                  <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_5277a9fd15328c03841df5827_e493e654e5" tabindex="-1" value=""></div>
```
Comment (lijiarui) [inline index.html]: Change name here

**Comment by huan** [comment]
It seems that you did not get my point at all ...

What I said is that you should understand how this repository works.

**Comment by lijiarui** [comment]
Well, to be honest,I'm not sure how this repo work because it doesn't looks like a static page and I cannot find any related  intro about this repo.

After read all of the code more, still not pretty sure, l guess you mean I should put all files in `bin` folder?

**Comment by huan** [comment]
Close this PR because it's already published at <https://chatie.github.io/www>

**Pull request closed without merging**